src/lambda-powertools-layer.ts

```diff
@@ -122,6 +127,84 @@ export class LambdaPowertoolsLayer extends lambda.LayerVersion {
   }
 }
 
+export class LambdaPowertoolsLayerPythonV3 extends lambda.LayerVersion {
+  /** CDK Layer for Python v3 **/
+  static constructBuildArgs(
+    includeExtras: boolean | undefined,
+    version: string | undefined,
+  ): string {
+    let suffix = '';
+    if (includeExtras) {
+      suffix = '[all]';
+    }
+    if (version) {
+      if (version.startsWith('git')) {
+        suffix = `${suffix} @ ${version}`;
+      } else {
+        suffix = `${suffix}==${version}`;
+      }
+    }
+    return suffix;
+  }
+
+  constructor(scope: Construct, id: string, props?: PowertoolsLayerProps) {
+    const languageName = 'Python';
+    const dockerFilePath = path.join(__dirname, `../layer/${languageName}v3`);
+    let runtimeFamilyPerVersion;
+    const compatibleArchitectures = props?.compatibleArchitectures ?? [
+      lambda.Architecture.X86_64,
+    ];
+    const compatibleArchitecturesDescription = compatibleArchitectures
+      .map((arch) => arch.name)
+      .join(', ');
+
+    if (props?.pythonVersion == '3.8') {
```

Comment:
What do you think of replacing the `pythonVersion` as a string and narrow it down to `Runtime.PYTHON_X`?

This would allow us:

```typescript
  new LambdaPowertoolsLayerPythonV3(stack, 'PowertoolsLayerPythonV3', {
    runtimeFamily: RuntimeFamily.PYTHON,
    runtime: Runtime.PYTHON_3_12, // only possible to set .PYTHON_X values
    compatibleArchitectures: [Architecture.X86_64],
  });
```

Reply:
I need to pass this string to the container, so, I'll need to cast this. Let me try.

Reply:
Done